packages/devextreme/js/__internal/filter_builder/m_filter_builder.ts

```diff
@@ -516,6 +516,15 @@ class FilterBuilder extends Widget<any> {
     }
   }
 
+  _getUniqueFieldIdentifier(item, currentCondition) {
+    const isDataFieldUnique = !this._model.some(
+      (condition) => condition !== currentCondition
+        && Array.isArray(condition)
+        && condition[0] === item.dataField,
+    );
+    return isDataFieldUnique ? item.dataField : item.name ? item.name : item.dataField;
```

Comment:
@tongsonbarbs please avoid nested ternary operators.

```
if (isDataFieldUnique) {
  return item.dataField;
}

return item.name ? item.name : item.dataField;
```

Reply:
Got it, thanks @siarhei-arzamasau

Reply:
https://github.com/DevExpress/DevExtreme/pull/28315/commits/1b85c6fdf74f75fce4ccdb1340307bbc9b483523